astropy/modeling/tests/example_models.py

```diff
@@ -289,5 +298,15 @@
         'x_lim': [-10, 10],
         'y_lim': [-10, 10],
         'integral': np.pi * (10 ** 2 - 5 ** 2)
+    },
+
+    Sersic2D: {
+        'parameters': [1, 25, 4, 50, 50, 0.5, -1],
+        'x_values': [0.0, 1, 10, 100],
+        'y_values': [1, 100, 0.0, 10],
+        'z_values': [1.686398e-02, 9.095221e-02, 2.341879e-02, 9.419231e-02],
+        'requires_scipy': True,
+        'x_lim': [-100,100],
+        'y_lim': [-100,100]
     }
-}
+}
```

Comment:
@cdeil, it's simple enough to create the temp variables you suggest. 
 "line-profile" as in memory profiling like we discussed Monday?